mev_inspect/block.py

```diff
@@ -25,6 +25,7 @@
 
 async def _get_logs_for_topics(after_block, before_block, topics):
     trials = 0
+    n_trials = 3
     while trials < 3:
```

Comment:
Consider using your newly created variable here:

```suggestion
    while trials < n_trials:
```

Reply:
:heavy_check_mark: